_authors/osci_2023/vnandi.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Vikram Nandi
+short_name: vnandi
+linkedin: https://www.linkedin.com/in/vikram-nandi/
```

Comment:
@dtaivpp Is this the correct format for LinkedIn?

Reply:
That is!